src/sql/workbench/contrib/notebook/browser/notebook.component.ts

```diff
@@ -136,6 +136,9 @@ export class NotebookComponent extends AngularDisposable implements OnInit, OnDe
 		this._register(DOM.addDisposableListener(window, DOM.EventType.KEY_DOWN, (e: KeyboardEvent) => {
```

Comment:
You should have the listener on the notebook component itself - this shouldn't need to be hooked up at the window level (and then you should be able to get rid of the isActive check too)

Reply:
Looked into this myself, we unfortunately need this for the time being.